src/sentry/integrations/aws_lambda/utils.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+
+
+# Taken from: https://gist.github.com/gene1wood/5299969edc4ef21d8efcfea52158dd40
+def parse_arn(arn):
```

Comment:
@scefali we should make tests for this function

Reply:
@NisanthanNanthakumar for sure, i'm going to add tests in a future PR